nginx/config/base.go

```diff
@@ -17,8 +17,13 @@ package config
 const BaseTemplate = `
 worker_processes 4;
 worker_rlimit_nofile 4096;
+{{if .is_windows}}
+pid C:/tmp/nginx.pid;
+{{else}}
 pid /tmp/nginx.pid;
+{{end}}
 user root root;
+error_log stderr;
 
 events {
   worker_connections 2048;
```

Comment:
Why such low connections?

Reply:
This was the default config it came with, it shouldn't really ever handle that many concurrent http requests anyway